hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java

```diff
@@ -553,11 +553,16 @@ protected void init720() {
 				.withType(ColumnTypeEnum.STRING, 500);
 
 		// Stop writing to hfj_forced_id https://github.com/hapifhir/hapi-fhir/pull/5817
-		Builder.BuilderWithTableName forcedId = version.onTable("HFJ_FORCED_ID");
-		forcedId.dropForeignKey("20240402.1", "FK_FORCEDID_RESOURCE", "HFJ_RESOURCE");
-		forcedId.dropIndex("20240402.2", "IDX_FORCEDID_RESID");
-		forcedId.dropIndex("20240402.3", "IDX_FORCEDID_TYPE_FID");
-		forcedId.dropIndex("20240402.4", "IDX_FORCEID_FID");
+
+		// --------------------------------------------------------------------------------------
+		// Removing these indexes before 7.6 breaks zero-downtime upgrades because performance of
+		// versions previous to 7.2 will degrade significantly without them.
```

Comment:
I worry about this change being merged back into the master branch.  Please add a comment to discourage this.
```suggestion
		// Removing these indexes before 7.6 breaks zero-downtime upgrades because performance of
		// versions previous to 7.2 will degrade significantly without them.
		// WARNING: do NOT merge this change back into the master branch.  These changes should only apply to these dead release branches.
```